scripts/cmake-configurator/parser/cmake_row.py

```diff
@@ -32,33 +35,33 @@ def __init__(self, raw_row: str, row_number: int):
       """
       row  = raw_row.strip()
       matches: Optional[Match[str]]  = re.match(self.regex_expression, row)
-      self.type: str =  self.NOT_PARSABLE
+      self.type: str =  types.NOT_PARSABLE.value
```

Comment:
**ISSUE**: What's the point of using an enum if than you're accessing its value? This is not OK. In particular for string values. The `type` field should keep an instance of the Enum class, that's the whole point of having an Enum, right? Also, this way the linter can do checks, otherwise any string would be acceptable.

Also, do you really need to convert the enum type to string somewhere? If you do, it's always possible with a simple dict comprehension expression to generate a value -> enum item string.

Reply:
Done, now code uses enums properly